### DISCUSSION TITLE
Warnings reading scanned-image info

### DISCUSSION THREAD

**Discussion by Jmuccigr** [ImageMagick/ImageMagick#5287] [Help]
Just starting using a new photo scanner (Epson V600) and when I use identify to grab the info, IM reports this error (using grep to get just the relevant line which describe the image width):
```
> identify -verbose Pictures/image.tif | grep -i 5584
  Geometry: 3998x5584+0+0
  Page geometry: 3998x5584+0+0
identify: Incorrect value for "RichTIFFIPTC"; tag ignored. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/960.
identify: Wrong data type 3 for "PixelXDimension"; tag ignored. `TIFFReadCustomDirectory' @ warning/tiff.c/TIFFWarnings/960.
identify: Wrong data type 3 for "PixelYDimension"; tag ignored. `TIFFReadCustomDirectory' @ warning/tiff.c/TIFFWarnings/960.
```

When I look up the tags, I find that they're perhaps needed only for compressed images (which this isn't). Is that what IM is complaining about?

exif spec is here: https://www.cipa.jp/std/documents/e/DC-X008-Translation-2019-E.pdf

Also, I find that exiftool seems to call this same tag by another name:
```
> exiftool -s Pictures/image.tif | grep -i 5584
ImageHeight                     : 5584
ExifImageHeight                 : 5584
ImageSize                       : 3998x5584
```

Looking into that.

Thanks for any help.

**Comment by snibgo** [comment]
Those are warnings, not errors.

**Reply by Jmuccigr** [reply]
> Those are warnings, not errors.

Yes, you are right; they are. I've edited the thread title to reflect that.
